firmware/controllers/engine_controller.cpp

```diff
@@ -580,7 +580,7 @@ void commonInitEngineController(Logging *sharedLogger DECLARE_ENGINE_PARAMETER_S
 #endif /* EFI_IDLE_CONTROL */
 
 #if EFI_ELECTRONIC_THROTTLE_BODY
-	initElectronicThrottle(PASS_ENGINE_PARAMETER_SIGNATURE);
+	//initElectronicThrottle(PASS_ENGINE_PARAMETER_SIGNATURE);
```

Comment:
why?

Reply:
because he was playing with the stepper, which used to use the ETB's IO config